float32 steering_ang
float32 collision_prob
